doc/examples/commctl_host_create.rst

```diff
@@ -1,4 +1,4 @@
 .. code-block:: shell
 
-   $ commctl host create 192.168.152.110 /path/to/priv/ssh_key
+   $ commctl host create --cluster my_cluster 192.168.152.110 /path/to/priv/ssh_key
```

Comment:
My wording isn't clear. It's a admin user ON the host in either case. I'll clarify.